### PR TITLE
CME-122: Database monitoring and alerting on RD repositories

### PR DESCRIPTION
**Pull request by SammiChong**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CME-247

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

**Review by lukasz-wolski — DISMISSED** [review]

**Review by rapolaskaseliscgi — APPROVED** [review]

**Comment by SammiChong** [comment]
Adding low thresholds to test emails are sending.

File: infrastructure/main.tf [hunk 1]
```diff
@@ -60,13 +61,18 @@ module "db-rd-caseworker-ref-v16" {
...
+  pgsql_version               = "16"
+  pgsql_sku                   = var.pgsql_sku
+  product                     = "rd"
+  name                        = local.db_name
+  pgsql_server_configuration  = var.pgsql_server_configuration
+  action_group_name           = join("-", [var.action_group_name, local.db_name, var.env])
+  email_address_key           = var.email_address_key
+  email_address_key_vault_id  = data.azurerm_key_vault.rd_key_vault.id
+  cpu_threshold = var.cpu_threshold
+  memory_threshold = var.memory_threshold
```
Comment (kiran-yenigala-hmcts) [inline infrastructure/main.tf:74]: please format this correctly

File: infrastructure/main.tf [hunk 2]
```diff
@@ -100,14 +106,20 @@ module "db-rd-caseworker-ref-v16-replica" {
...
+  product                        = "rd"
+  name                           = join("-", [var.product-v16, var.component-v16, "replica"])
+  resource_group_name            = "rd-caseworker-ref-api-postgres-db-v16-data-${var.env}"
+  create_mode                    = "Replica"
+  source_server_id               = var.primary_server_id
+  pgsql_server_configuration     = var.pgsql_server_configuration
+  action_group_name              = join("-", [var.action_group_name, local.db_name, "replica", var.env])
+  email_address_key              = var.email_address_key
+  email_address_key_vault_id     = data.azurerm_key_vault.rd_key_vault.id
+  cpu_threshold = var.cpu_threshold
```
Comment (kiran-yenigala-hmcts) [inline infrastructure/main.tf:120]: please do the formatting

**Review by AntonyLeons — DISMISSED** [review]

**Review by kiran-yenigala-hmcts — APPROVED** [review]
LGTM

**Review by lukasz-wolski — APPROVED** [review]

**Review by SabinaHMCTS — APPROVED** [review]

**Pull request merged**